lxd/api.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/lxc/lxd/lxd/cluster/request"
 	"github.com/lxc/lxd/lxd/db"
 	"github.com/lxc/lxd/lxd/project"
+	lxdrequest "github.com/lxc/lxd/lxd/request"
```

Comment:
Oh can you call this `lxdRequest` thanks

Reply:
Sure thing